crates/pop-contracts/src/lib.rs

```diff
@@ -15,7 +15,7 @@ pub use call::{
 	call_smart_contract, dry_run_call, dry_run_gas_estimate_call, set_up_call, CallOpts,
 };
 pub use new::{create_smart_contract, is_valid_contract_name};
-pub use templates::{ContractType, Template};
+pub use templates::{ContractTemplate, ContractType};
```

Comment:
As its in the templates namespace, can we not just have `templates::{Contract, Type}` ?